lib/plugins/aws/provider.js

```diff
@@ -480,6 +481,14 @@ class AwsProvider {
             required: ['Fn::Sub'],
             additionalProperties: false,
           },
+          awsCfBase64: {
+            type: 'object',
+            properties: {
+              'Fn::Base64': {
+                anyOf: [{ const: '' }, { $ref: '#/definitions/awsCfInstruction' }],
```

Comment:
Maybe we can define it simpler as  `'Fn::Base64': {}` ? It'll mean we will allow any value, and let AWS do stricter validation